### ISSUE TITLE
feat: qrAutoKill

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2351]
Kill the process after x qr codes.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "minor"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.27.0](https://github.com/open-wa/wa-automate-nodejs/compare/4.26.1...4.27.0) (2021-12-16)

- Release 4.27.0 [`76b17a2`](https://github.com/open-wa/wa-automate-nodejs/commit/76b17a267951c828d6b04276de88cd6e2a226bf2)
- ✨ feat: `config.qrMax` `--qr-max` #2351 [`8b6c268`](https://github.com/open-wa/wa-automate-nodejs/commit/8b6c2682e079628700bfde9b3a12a2cd03e0b65c)
- 🔧 fix config parse order #2271 [`1721726`](https://github.com/open-wa/wa-automate-nodejs/commit/17217264d7916f1e92621e75c0795f37be748e00)
- 🔊 migrate from console.log to spinner for `qrMax` #2351 [`ab52fc0`](https://github.com/open-wa/wa-automate-nodejs/commit/ab52fc0017c5d21fd85312f9c0c49ca00e74f0d5)
- updated types-only package [`8ace227`](https://github.com/open-wa/wa-automate-nodejs/commit/8ace227434c66478e3150e5b92209ec06c1bdbf3)
